components/lvgl_ili9341/ili9488.c

```diff
@@ -113,8 +113,9 @@ void ili9488_flush(lv_disp_drv_t * drv, const lv_area_t * area, lv_color_t * col
 {
     uint32_t size = lv_area_get_width(area) * lv_area_get_height(area);
 
-// #define 24BIT_DATA
-#ifdef 24BIT_DATA
+#define DATA_24BITS
+
+#if defined DATA_24BITS
```

Comment:
The code associated with DATA_24BITS does not work please remove it

Reply:
Only RGB 565 to RGB 666 works

Reply:
Addressed in e8224db